R/agg_scale.R

```diff
@@ -83,6 +88,15 @@
 #' location-years have 5-year age groups while other location-years have 1-year
 #' age groups.
 #'
+#' In addition it is okay if the interval variables included in `id_vars` are
```

Comment:
Does this work for categorical variables or just intervals? For example, if you have male & female for some locs and both-sex for other locs can you aggregate a location hierarchy? In this example m&f would be aggregated to both-sex before location-hierarchy aggregation.

Reply:
Just intervals. So in that example it would aggregate each categorical variable separately up the location hierarchy